aws/table_aws_cloudfront_cache_policy.go

```diff
@@ -0,0 +1,176 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudFrontCachePolicy(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_cloudfront_cache_policy",
+		Description: "AWS CloudFront Cache Policy",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"NoSuchCachePolicy"}),
+			Hydrate:           getCloudFrontCachePolicy,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listCloudFrontCachePolicies,
+		},
+		Columns: awsRegionalColumns([]*plugin.Column{
```

Comment:
This is not a regional resource, I don’t think we need to use `awsRegionalColumns`, please check .